xwiki-mysql-tomcat/Dockerfile

```diff
@@ -32,12 +32,18 @@ ENV XWIKI_VERSION=8.4.4 \
 RUN echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list && \
   apt-get update && \
   apt-get -y upgrade && \
-  apt-get -y --force-yes install sudo nano unzip curl openjdk-8-jdk tomcat8 libreoffice
+  apt-get --no-install-recommends -y --force-yes install sudo nano unzip curl openjdk-8-jdk tomcat8 libreoffice && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
BTW at some time I was doing a "apt-get clean" but apparently it got lost. Maybe it's nicer than the rm (are they doing the same thing)?